### PR TITLE
Ewl 1102 reduce colors

### PR DESCRIPTION
**Pull request by degouville**
# About the type of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Test plan

- [ ] Empty data
- [ ] Invalid data
- [ ] Correct data

# Checklist
### Before code review

- [ ] Move Jira ticket into PR
- [ ] Check with another developers that features sounds good

### After code review

- [ ] Move Jira ticket into QA

### After QA validation
If conflicts are present in github:
  - [ ] `git pull --rebase origin master`
  - [ ] Fix conflicts
  - [ ] `git rebase --continue` repeat until there is no more conflicts
  - [ ] `git push --force-with-lease`

If there's no conflicts
- [ ] squash and merge
- [ ] delete branch

### CODE REVIEW

File: src/elements/Colors/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+// eslint-disable-next-line react/prop-types
+const renderColor = ({ option, ...color }) => {
+  const [[name, hex]] = Object.entries(color);
+  const isLegacy = name.includes('_');
+
+  // TODO: Remove '❌' after QA
+  return (
+    <StyledColor hex={hex} option={option}>
+      {isLegacy && '❌'}
```
Comment (Xb0X) [inline src/elements/Colors/index.js]: Love it :D !!

File: src/elements/LiveLabel/index.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ export default styled.div`
   justify-content: center;
   padding: 5px 8px;
   border-radius: 2px;
-  background-color: ${colors.actionTwoDarkBase};
-  color: ${colors.coreLightMinus1};
+  background-color: ${colors.monza};
```
Comment (davidgarry) [inline src/elements/LiveLabel/index.js]: It should be `utahCrimson`

File: src/modules/BurgerMenu/RightMenu/RightMenuItem.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import styled, { css } from 'react-emotion';
 import kebabCase from 'lodash/kebabCase';
 import { medium, large } from '../../../breakpoints';
-import { arsenic, cerulean, regentGray, turquoiseBlue, whiteLilac } from '../../../colors';
+import { mako, cerulean, regentGray, turquoiseBlue, athensGray } from '../../../colors';
```
Comment (davidgarry) [inline src/modules/BurgerMenu/RightMenu/RightMenuItem.js]: `whiteLilac should be replaced by `blackHaze` with code `e8e8eb`

File: src/theme.js [hunk 4]
```diff
@@ -1,197 +1,192 @@
...
   link: {
-    color: featureOneMinus2,
+    color: malibu,
   },
   button: {
     primary: {
-      color: coreLightMinus1,
-      backgroundColor: actionTwoDarkBase,
+      color: white,
+      backgroundColor: monza,
```
Comment (davidgarry) [inline src/theme.js]: It should be `utahCrimson`

**Comment by degouville** [comment]
Thanks @Xb0X ;)

File: src/colors.js [hunk 5]
```diff
@@ -1,117 +1,68 @@
...
 export const paleSky = '#6D777F';
+export const regentGray = '#798490';
+export const dustyGray = '#979797';
+export const stormGray = '#6D6D78';
+export const scarpaFlow = '#4C4B4F';
+export const mako = '#3D4248';
+export const gunPowder = '#41435B';
+export const martinique = '#2D304A';
+export const ebonyClay = '#22263C';
 export const blackPearl = '#050618';
```
Comment (davidgarry) [inline src/colors.js]: Can you use the code from brandPlus2 for blackPearl : `#060825`

Comment (degouville) [reply src/colors.js]: ✅

File: src/theme.js (hunk 4 above)
Comment (degouville) [reply src/theme.js]: ✅

File: src/modules/BurgerMenu/RightMenu/RightMenuItem.js (hunk 3 above)
Comment (degouville) [reply src/modules/BurgerMenu/RightMenu/RightMenuItem.js]: ✅

File: src/elements/LiveLabel/index.js (hunk 2 above)
Comment (degouville) [reply src/elements/LiveLabel/index.js]: ✅

File: src/components/Quickpoll/index.js [hunk 6]
```diff
@@ -64,7 +64,7 @@ const StyledChoice = styled(Button)`
       padding: 2px 0;
       border: 0px;
       border-radius: 0;
-      background-color: ${colors.actionOneDarkMinus1};
+      background-color: ${colors.malibu};
```
[the remark this answers is not in the document]
Comment (degouville) [reply src/components/Quickpoll/index.js]: ✅

File: src/components/Quickpoll/index.js [hunk 7]
```diff
@@ -8,10 +8,10 @@ import Spinner from '../../elements/Spinner';
 
 const StyledContainer = styled.div`
   border-radius: 4px;
-  background-color: ${colors.flawlessMahogany};
+  background-color: ${colors.gunPowder};
```
[the remark this answers is not in the document]
Comment (degouville) [reply src/components/Quickpoll/index.js]: ✅

File: src/components/Card/WatchbarCard.js [hunk 8]
```diff
@@ -114,7 +114,7 @@ const StyledContent = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  background: ${colors.actionTwoDarkBase};
+  background: ${colors.monza};
```
[the remark this answers is not in the document]
Comment (degouville) [reply src/components/Card/WatchbarCard.js]: ✅

**Pull request merged**